src/app/app.tsx

```diff
@@ -9,22 +10,31 @@ import { BiometryAvailabilityProvider } from '../biometry/biometry-availability.
 import { RootStackScreen } from '../navigator/root-stack';
 import { persistor, store } from '../store/store';
 import { ToastProvider } from '../toast/toast-provider';
+import { HideBalanceProvider } from '../utils/hide-balance/hide-balance.provider';
+import { initQuickActions } from '../utils/quick-actions.utils';
 import { initSentry } from '../utils/sentry.utils';
 
 initSentry();
 enableScreens();
+initQuickActions();
 
 export const App = () => {
-  useEffect(() => void hide(), []);
+  useEffect(() => {
+    void hide();
+
+    return () => QuickActions.clearShortcutItems();
```

Comment:
that's weird that you `setShortcutItems ` outside the component life cycle and calling `clearShortcutItems` on unmount